opensquirrel/parsing/antlr/generated/CQasm3Parser.py

```diff
@@ -1030,6 +1030,8 @@ def exitRule(self, listener: ParseTreeListener):
 
         def accept(self, visitor: ParseTreeVisitor):
             if hasattr(visitor, "visitGateApplication"):
+                if "measure" in str(self.children[0]).lower():
```

Comment:
We should not make any changes to any of the modules in the `opensquirrel/parsing/antlr/...` directory. We should remove all the calls to the antlr parser in the tests and replace them with the libqasm parser, where appropriate.